tests/engines/test_pil.py

```diff
@@ -24,6 +24,11 @@
 except ImportError:
     METADATA_AVAILABLE = False
 
+try:
+    from unittest import mock  # Python 3.3 +
```

Comment:
@kkopachev We don't need python 3 support right now, I think it would just inflate our code base.